src/components/pages/Cases/CaseTable.js

```diff
@@ -267,59 +275,56 @@ export default function CaseTable(props) {
       ),
     },
   ];
-  const findRowByID = (selectedRowID, caseData) => {
-    caseData.forEach(row => {
-      if (row.case_id === selectedRowID) {
-        return row;
-      } else {
-        return 'Please select row to save.';
+
+  const findRowByID = (rowID, rowData) => {
+    for (let i = 0; i < rowData.length; i++) {
+      let currentRow = rowData[i];
+
+      if (currentRow.case_id === rowID) {
+        return currentRow;
       }
-    });
+    }
+    return 'Please select case to save.';
   };
 
```

Comment:
I also implemented the loop for the code. Glad you caught that as well! Only thing that differs is our exit message.